### PR TITLE
Fixes lp#1728559: Updated help to reflect new world.

### PR DESCRIPTION
**Pull request by anastasiamac**
## Description of change

Help on show-offer was outdated. This PR updates help to reflect current reality.

## QA steps

'juju help show-offer' makes sense and should look like this: https://pastebin.canonical.com/202240/

## Bug reference

https://bugs.launchpad.net/juju/+bug/1728559

### CODE REVIEW

File: cmd/juju/crossmodel/show.go [hunk 1]
```diff
@@ -14,16 +14,30 @@ import (
...
 
 Examples:
-   $ juju show-offer fred/prod.db2
-   $ juju show-offer anothercontroller:fred/prod.db2
+To show the extended information for the application 'prod' offered
+from the model 'default' on the same Juju controller:
+
+     juju show-offer default.prod
+
+The supplied URL can also include a username where offers require them. This will be given as part of the URL retrieved from the
```
Comment (jameinel) [inline cmd/juju/crossmodel/show.go]: we seem to be word wrapping the above paragraphs, should we do so here as well?

File: cmd/juju/crossmodel/show.go [hunk 2]
```diff
@@ -14,16 +14,30 @@ import (
...
+'juju find-offers' command. To show information for the application
+'prod' from the model 'default' as the user 'admin':
```
Comment (jameinel) [inline cmd/juju/crossmodel/show.go]: I didn't think this was *as* the user admin, this was *from* the user admin.
It still shows you the offer as seen by your login. It just allows you to see offers from models that other people are managing.

**Comment by anastasiamac** [comment]
$$merge$$

**Comment by anastasiamac** [comment]
$$merge$$

**Pull request merged**